.github/workflows/nightly.yml

```diff
@@ -373,8 +375,45 @@ jobs:
           echo opcache.jit_hot_func=1 >> /etc/php.d/opcache.ini
           echo opcache.jit_hot_return=1 >> /etc/php.d/opcache.ini
           echo opcache.jit_hot_side_exit=1 >> /etc/php.d/opcache.ini
+          echo opcache.jit_blacklist_root_trace=255 >> /etc/php.d/opcache.ini
+          echo opcache.jit_blacklist_side_trace=255 >> /etc/php.d/opcache.ini
+          echo opcache.file_update_protection=0 >> /etc/php.d/opcache.ini
+          echo opcache.memory_consumption=256M >> /etc/php.d/opcache.ini
+          echo opcache.interned_strings_buffer=64 >> /etc/php.d/opcache.ini
+          echo opcache.max_accelerated_files=100000 >> /etc/php.d/opcache.ini
           echo memory_limit=-1 >> /etc/php.d/opcache.ini
           php -v
+      - name: Test Infection
```

Comment:
I think your PRs still do too much at a time :slightly_smiling_face: E.g. just adding Infection without `jit_check.php` or other changes would be much easier to review and merge.